src/api/personal.rs

```diff
@@ -81,7 +81,7 @@ impl<T: Transport> Personal<T> {
     /// Imports a raw key and protects it with the given password.
     /// Returns the address of created account.
     pub fn import_raw_key(&self, private_key: &[u8; 32], password: &str) -> CallFuture<Address, T::Out> {
-      let private_key = hex::encode(private_key);
+        let private_key = hex::encode(private_key);
         let private_key = helpers::serialize(&private_key);
```

Comment:
Is it idiomatically acceptable to shadow variables repeatedly like this? I'm fine with it, just curious about the general community expectations.

Reply:
Yes, they are actually not shadowed, but rather re-declared (previous one can be dropped) and afaik it's totally acceptable pattern.